makeflow/src/mf_mesos_executor

```diff
@@ -0,0 +1,137 @@
+import os
```

Comment:
I added the detailed explanatory comment about the architecture in `batch_job/src/batch_job_mesos.c` and added brief 
explanatory of each module in `mf_mesos_scheduler`, `mf_mesos_executor`, `mf-mesos-executor` and `mf_mesos_setting` respectively.